lib/ansible/modules/storage/netapp/na_ontap_aggregate.py

```diff
@@ -155,32 +182,41 @@ class NetAppOntapAggregate(object):
     def __init__(self):
         self.argument_spec = netapp_utils.na_ontap_host_argument_spec()
         self.argument_spec.update(dict(
-            state=dict(required=False, choices=[
```

Comment:
By changing the order of the arguments it's hard to review what's changed. Please be mindful of cosmetic changes

Reply:
I can change this back if you want.

Reply:
I would not change it now, but I agree this was painful for me to review this internally as well.
So let's try to avoid gratuitous changes and communicate this to the group.

Reply:
@gundalow do you want us to change it back for this request? or just in the future not make these kind of cosmetic changes?

Reply:
If you could change that would make it easier.
Making life easier for the review is one of the things that will help speed up merges for you long term :)

Reply:
@gundalow I'm not sure I'll be able to. 

All changes we made used to go through a long internal legal/security approval process which requires us to the group are internal PR together before it can go through the process and then allowed to be pushed out to Github. We recently got approval that we can push out content as it completed which should help in the future but that won't help for any of the work we've done over the last few months. As everything had been merged together and tested that way expecting to go through the old process. 
In this case, there are 5 different internal PR in this request have already been merged together, which there isn't an easy way to separate out.